code/modules/clothing/masks/bandana.dm

```diff
@@ -200,3 +200,35 @@
 	desc = "A fine black bandana with nanotech lining and a skull emblem."
 	greyscale_colors = "#2e2e2e#C6C6C6"
 	flags_1 = NONE
+
+/obj/item/clothing/mask/bandana/facescarf
+	name = "facescarf"
+	desc = "Cover your face like in the cowboy movies. It also has breathtube so you can wear it everywhere!"
+	actions_types = /datum/action/item_action/adjust
+	icon_state = "facescarf"
+	inhand_icon_state = "greyscale_facescarf"
+	alternate_worn_layer = BACK_LAYER
+	clothing_flags = BLOCK_GAS_SMOKE_EFFECT|MASKINTERNALS
+	flags_inv = HIDEFACIALHAIR | HIDEFACE | HIDESNOUT
+	w_class = WEIGHT_CLASS_SMALL
+	visor_flags = BLOCK_GAS_SMOKE_EFFECT | MASKINTERNALS
+	visor_flags_inv = HIDEFACIALHAIR | HIDEFACE | HIDESNOUT
+	flags_cover = MASKCOVERSMOUTH
+	visor_flags_cover = MASKCOVERSMOUTH
+	custom_price = PAYCHECK_CREW
+	tint = 0
+	greyscale_colors = "#eeeeee"
+	greyscale_config = /datum/greyscale_config/facescarf
+	greyscale_config_worn = /datum/greyscale_config/facescarf/worn
+	greyscale_config_inhand_left = /datum/greyscale_config/facescarf/inhands_left
+	greyscale_config_inhand_right = /datum/greyscale_config/facescarf/inhands_right
+	flags_1 = IS_PLAYER_COLORABLE_1
```

Comment:
```suggestion
/obj/item/clothing/mask/bandana/facescarf
	name = "facescarf"
	desc = "Cover your face like in the cowboy movies. It also has a breathtube so you can wear it everywhere!"
	actions_types = /datum/action/item_action/adjust
	icon_state = "facescarf"
	inhand_icon_state = "greyscale_facescarf"
	alternate_worn_layer = BACK_LAYER
	clothing_flags = BLOCK_GAS_SMOKE_EFFECT|MASKINTERNALS
	w_class = WEIGHT_CLASS_SMALL
	visor_flags = BLOCK_GAS_SMOKE_EFFECT | MASKINTERNALS
	custom_price = PAYCHECK_CREW
	greyscale_colors = "#eeeeee"
	greyscale_config = /datum/greyscale_config/facescarf
	greyscale_config_worn = /datum/greyscale_config/facescarf/worn
	greyscale_config_inhand_left = /datum/greyscale_config/facescarf/inhands_left
	greyscale_config_inhand_right = /datum/greyscale_config/facescarf/inhands_right
```

All you need to do is remove the shared variables (and tint, which should already be 0). I would also check to see if the AltClick and examine procs still work the way you expect them to in-game